_includes/head.html

```diff
@@ -34,6 +34,10 @@
     <script src="/javascripts/jquery.loadTemplate-1.4.4.min.js"></script>
     <script src="/javascripts/respond.js"></script>
     <script src="/javascripts/ua-parser.min.js"></script>
+    <script src="https://npmcdn.com/imagesloaded@4.1/imagesloaded.pkgd.min.js"></script>
```

Comment:
should this also be inside the `if page.include_masonry` or does it do other things?